core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/upgrade/extensions/replication/repmgr-for-pg-10-upgrade.yml

```diff
@@ -0,0 +1,136 @@
+---
+# Documentation: https://repmgr.org/docs/5.2/upgrading-repmgr-extension.html#UPGRADING-MAJOR-VERSION#
+# Compatibility: https://repmgr.org/docs/5.2/install-requirements.html#INSTALL-COMPATIBILITY-MATRIX
+
+# Expects:
+#   - pg_service_names [dict] (list for each OS family)
+
+- name: repmgr for PG 10 | Create upgrade state file
+  file:
+    path: "{{ upgrade.repmgr.upgrade_state_file_path }}"
+    state: touch
+    mode: u=rw,g=r,o=
+
+- name: repmgr for PG 10 | Load postgresql role manifest
+  include_vars:
+    file: manifest.yml
+    name: postgresql_manifest
+
+- name: repmgr for PG 10 | Search for primary node
+  become_user: postgres
+  # command prints primary node name (hostname)
+  shell: |-
+    set -o pipefail && \
+    {{ pg_10.repmgr.bin_dir[ansible_os_family] }}/repmgr cluster show \
+      | awk 'BEGIN{FS="|"} {gsub(/ /,""); if ($3 == "primary") print $2}'
+  changed_when: false
+  register: find_pg_primary_node
+  failed_when: find_pg_primary_node.rc != 0 or find_pg_primary_node.stdout == ""
+  run_once: true
+  args:
+    executable: /bin/bash
+
+# Step: Stop repmgr service
+- name: repmgr for PG 10 | Stop repmgr service
+  systemd:
+    name: "{{ pg_10.repmgr.service_name[ansible_os_family] }}"
```

Comment:
Naming of such variables is in many places and they should be version independent.

Reply:
Refactored 👍🏻